Documentation/TopLevelObjects/Config.rst

```diff
@@ -503,18 +503,34 @@ Properties of 'config'
         comment in the <head> section of the page. Use this to insert a note
         like that "Programmed by My-Agency".
 
-    ..  confval:: htmlTag.attributes
+    ..  confval:: htmlTag.attributes.[attribute]
         :name: config-htmlTag-attributes
-        :type: array
+        :type: string /
         :Example: setup-config-htmltag-attributes
 
+        ..  versionchanged:: 14.0
+            It is now possible to use `stdWrap <https://docs.typo3.org/permalink/t3tsref:stdwrap>`_
+            options like :ref:`userFunc <t3tsref:confval-parsefunc-userfunc>`,
+            :ref:`override <t3tsref:confval-stdwrap-override>` or
+            `Data / getText <https://docs.typo3.org/permalink/t3tsref:data-type-gettext>`_
+            via TypoScript.
+
         Sets the attributes for the :html:`<html>` tag on the page. Allows to override
         and add custom attributes via TypoScript without having to re-add the
         existing attributes generated by SiteHandling.
 
         This property supersedes the previous :typoscript:`config.htmlTag_setParams` option by providing
         a more flexible API to add attributes.
 
+        ..  code-block:: typoscript
+
+            config.htmlTag.attributes.my-attribute = 123
+
+        ..  code-block:: typoscript
+
+            config.htmlTag.attributes.my-attribute = 123
+            config.htmlTag.attributes.my-attribute.userFunc = MyVendor\\MyExtension\\HtmlTagEnhancer->overrideMyAttribute
```

Comment:
Are the double backslashes needed your documentation? Escaping? In real TS I have never used them.

Reply:
I took them from the code example in the changelog. I have seen them before in TS. They are not needed by rst rendering